src/main/java/com/google/sps/api/authorization/AuthorizationRequester.java

```diff
@@ -43,7 +43,7 @@
    */
   public static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
   public static final String CLIENT_SECRETS_PATH = "/client_secrets.json";
-  public static final String OFFLINE_ACCESS_TYPE = "offline";
+  public static final String OFFLINE_ACCESS_TYPE = "online";
```

Comment:
I don't understand this change. If this has to be changed, then probably the name of the variable should too.

Reply:
This is a setting that we originally just used as it is used in the sample, but according to the docs, it should be "online" in case of online applications. (https://googleapis.dev/java/google-api-client/latest/com/google/api/client/googleapis/auth/oauth2/GoogleAuthorizationCodeFlow.Builder.html#setAccessType-java.lang.String-)

Reply:
makes sense. Then I guess we should rename it to `ACCESS_TYPE`

Reply:
Then please rename constant name to `ACCESS_TYPE`.